### DISCUSSION TITLE
V3 Linux Window Transparency

### DISCUSSION THREAD

**Discussion by mattwcole** [wailsapp/wails#3386] [General]
Transparent windows appear to have a black background on Linux in the V3 alpha. Linux is also skipped in the example [here](https://github.com/wailsapp/wails/blob/7c921d6a8128aad00168d7af2facf247bef3bfc3/v3/examples/window/main.go#L157). Is there a technical limitation, or has the feature simply not been implemented yet?

PS: Thanks for the great project!

**Comment by leaanthony** [comment]
Hi there! Thanks for this feedback.
Whilst v3 is in alpha, we have a strict feedback procedure outlined here: https://v3alpha.wails.io/getting-started/feedback
This is to reduce noise whilst the branch is unstable and keep all the information in one place. We'd love to have you as part of the [v3 feedback](https://discord.gg/BrRSWTaxVK) loop so please consider opening a new post there. Just a copy and paste would be fine.
Closing this for now.
Thanks again 🙏
